Lib/colorsys.py

```diff
@@ -151,16 +151,7 @@ def hsv_to_rgb(h, s, v):
     q = v*(1.0 - s*f)
     t = v*(1.0 - s*(1.0-f))
     i = i%6
-    if i == 0:
-        return v, t, p
-    if i == 1:
-        return q, v, p
-    if i == 2:
-        return p, v, t
-    if i == 3:
-        return p, q, v
-    if i == 4:
-        return t, p, v
-    if i == 5:
-        return v, p, q
+
+    l = [(v, t, p), (q, v, p), (p, v, t), (p, q, v), (t, p, v), (v, p, q)]
+    return l[i]
     # Cannot get here
```

Comment:
```suggestion
```

since there is nothing to remind of anymore.

Reply:
yes, changed it.